cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRollingTest.java

```diff
@@ -14,40 +17,55 @@
 
 public class RackRollingTest {
 
-
     @Test
     public void shouldRestartNoBrokersIfNoReason() throws ExecutionException, InterruptedException, TimeoutException {
         var brokerId = 0;
         RollClient client = mock(RollClient.class);
 
         when(client.isNotReady(brokerId)).thenReturn(false);
 
-        //Assuming the pods is Serving
         doReturn(State.SERVING).when(client).observe(brokerId);
         RackRolling.rollingRestart(client, asList(brokerId), new Function<Integer, Set<RestartReason>>() {
             @Override
             public Set<RestartReason> apply(Integer integer) {
                 return Set.of();
             }
-        }, 1000, 1000, 1, 1);
+        }, Reconciliation.DUMMY_RECONCILIATION, KafkaVersionTestUtils.getLatestVersion(), new Function<Integer, String>() {
+            @Override
+            public String apply(Integer integer) {
+                return "";
+            }
```

Comment:
We're going to be writing a lot of tests and many of them won't be testing the dynamic config stuff, so we can factor out some constant `EMPTY_CONFIG_SUPPLIER`